src/d/actor/d_a_tag_kk1.cpp

```diff
@@ -3,64 +3,120 @@
 // Translation Unit: d_a_tag_kk1.cpp
 //
 
+
 #include "d/actor/d_a_tag_kk1.h"
+#include "d/d_com_inf_game.h"
 #include "d/d_procname.h"
 
-/* 000000EC-00000120       .text __ct__15daTag_Kk1_HIO_cFv */
```

Comment:
Let's keep this automatically generated comments in, the mangled symbol name is convenient for quick searches